docs/quickstart.rst

```diff
@@ -54,6 +54,8 @@ enabled, so you will need to
 - set the ``GENOMEKIT_GCS_BILLING_PROJECT`` env var to your Google Cloud project::
 
     export GENOMEKIT_GCS_BILLING_PROJECT="your-project-id"
+    # required depending on your local gcloud configuration
+    export GOOGLE_CLOUD_PROJECT="your-project-id"
```

Comment:
do you need the `GENOMEKIT_GCS_BILLING_PROJECT` then?

Reply:
Yes, that one is required for DefaultDataManager.
I've seen one case where GOOGLE_CLOUD_PROJECT was required, but I couldn't figure out why. I figured it's better to mention it here in case someone else also needs it.

Reply:
Could we just use GOOGLE_CLOUD_PROJECT internally? when would you have them pointing to two different values?

Reply:
I'm not sure if GOOGLE_CLOUD_PROJECT has some other undesired side effect.

In the case I observed, the error message was 
> Project was not passed and could not be determined from the environment.

Technically it could be a different project than the one used for billing the requester-pays egress charge. I don't know how to state that here without adding too much noise to the doc.